<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject{background-color: #3b5998;}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>700</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(83, 86, 173);
border: 5px solid white;
margin: 20% 0;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-20</y>
      <width>300</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{background-color: rgb(255, 255, 255);}
QLineEdit{
	
border: 2px solid #6b6b6b ;
  border-radius: 3px;
 padding: 10px 20px;
	background-color: rgb(255, 255, 255);}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>389</y>
       <width>120</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <fontweight>Black</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
    border: 0 none;
    border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}


QPushButton::hover{
   border: 2px solid white;
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
     border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}
</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>110</y>
       <width>209</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>First Name..</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>180</y>
       <width>209</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Last Name..</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>337</y>
       <width>120</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <fontweight>Black</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    width: 100px;
    background-color: rgba(83, 86, 173,0.4);
    font-weight: bold;
    color: white;
    border: 0 none;
     border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}


QPushButton::hover{
   border: 2px solid white;
    width: 100px;
      background-color: rgba(83, 86, 173,0.4);
    font-weight: bold;
    color: white;
     border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}
</string>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>247</y>
       <width>209</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Birth date..</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border:none;</string>
     </property>
     <property name="text">
      <string>     PERSONAL DETAILS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>91</y>
     <width>241</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color:#6b6b6b;
border:none;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>661</x>
     <y>70</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color: rgb(77, 77, 77);
border-radius:25px;
color: rgb(0, 0, 0);
border: none;
}
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(77, 77, 77);
border-radius:25px;
color: rgb(0, 0, 0);
/*border:none;*/
}
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{background-color: #ee0979;
border-radius:25px;
color: white;
border: 1px solid white;
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>91</y>
     <width>241</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:#6b6b6b;
border:none;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>126</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PERSONAL DETAILS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>126</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DOCUMENTS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>126</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ACCOUNT SETUP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
